appveyor.yml

```diff
@@ -1,6 +1,6 @@
 version: 3.2.0.{build}
```

Comment:
Let's submit this change separately, as it is orthogonal to the main change

Reply:
that was a last test (before going to bed ;) ) to gather information, not planned to be merged...